package.json

```diff
@@ -11,8 +11,15 @@
   "scripts": {
     "demo:pd": "IS_DEMO=true REACT_APP_API_URL=http://localhost:3001 yarn dev:pd",
     "dev:api": "yarn workspace spotlight-api run dev",
-    "dev:pd": "run-p dev:api dev:pdclient",
-    "dev:pdclient": "yarn workspace public-dashboard-client run dev"
+    "dev:pd": "run-p dev:api dev:pdc",
+    "dev:pdc": "yarn workspace public-dashboard-client run dev",
+    "dev:sc": "yarn workspace spotlight-client run dev",
+    "demo:spotlight": "IS_DEMO=true REACT_APP_API_URL=http://localhost:3001 yarn dev:spotlight",
+    "dev:spotlight": "run-p dev:api dev:sc",
+    "test": "run-p \"test:* -- {@}\" --",
+    "test:api": "yarn workspace spotlight-api run test",
+    "test:pdc": "yarn workspace public-dashboard-client run test",
+    "test:sc": "yarn workspace spotlight-client run test"
```

Comment:
I could not get this to run successfully without prepending `REACT_APP_API_URL=http://localhost:3001` to the test script in spotlight-client/package.json. For some reason it is not picking up that env var. Something in our environments must be configured differently. We should probably try to figure out what that is and document the setup.

Reply:
interesting. I have an `.env.test` file in this package that sets `REACT_APP_API_URL=http://localhost:3002` -- that may be what you are missing? the readme _kinda_ covers this but maybe it is not explicit enough?

Reply:
I am also seeing that the current .env requirements have drifted from our shared configuration in 1password ... i'm going to update that as well as the README to try to make it less enigmatic. sorry about this but thank you for calling it out!

Reply:
Oh yes I see it now in the spotlight-client readme. Adding a .env.test fixed it.